GVFS/GVFS.Common/NuGetUpgrade/NuGetUpgrader.cs

```diff
@@ -117,6 +117,13 @@ public class NuGetUpgrader : ProductUpgrader
             if (!upgraderConfig.IsReady(out error))
             {
                 return false;
+            }
+
+            string gitBinPath = GVFSPlatform.Instance.GitInstallation.GetInstalledGitBinPath();
+            if (string.IsNullOrEmpty(GVFSPlatform.Instance.GitInstallation.GetInstalledGitBinPath()))
+            {
+                error = $"NuGetUpgrader: Unable to locate git installation. Ensure git is installed and try again.";

```

Comment:
Should we add a timestamp?

Reply:
I thought about this - but we don't usually include timestamps with our error messages, and since we will have local logs, the timestamps will be present there. If we expected to not have logs, then I could see the making an exception from our general pattern to include a timestamp... Open to other thoughts on this, though.